src/meta/client/MetaClient.cpp

```diff
@@ -60,17 +59,17 @@ MetaClient::~MetaClient() {
 
 
 bool MetaClient::isMetadReady() {
-    if (sendHeartBeat_) {
-        auto ret = heartbeat().get();
-        if (!ret.ok() && ret.status() != Status::LeaderChanged()) {
-            LOG(ERROR) << "Heartbeat failed, status:" << ret.status();
-            ready_ = false;
-            return ready_;
-        }
-    }  // end if
-    loadData();
-    if (!FLAGS_local_config) {
-        loadCfg();
+    auto ret = heartbeat().get();
+    if (!ret.ok() && ret.status() != Status::LeaderChanged()) {
+        LOG(ERROR) << "Heartbeat failed, status:" << ret.status();
+        ready_ = false;
+        return ready_;
+    }
+
+    bool ldRet = loadData();
+    bool lcRet = loadCfg();
```

Comment:
When `FLAGS_local_config` is true, we don't load config from meta any more, this mode is used for local test (optimize configs).

Reply:
done!